### ISSUE TITLE
GDrive: Cannot preview spreadsheet from list view and grid view

### ISSUE THREAD

**Issue by cpiosecny** [adobe/helix-sidekick-extension#743]
**Description**
The changes in #725  seem to be insufficient for detecting the correct file type and thus the URL generated for the preview POST is wrong.

**To Reproduce**
1. Go to a folder in gdrive
2. Create a new spreadsheet and save it
3. Go to your file list and try to preview the spreadsheet
![image](https://github.com/adobe/helix-sidekick-extension/assets/16552927/0ddc557d-6607-4ec2-80d8-0bb387e2fb24)
the extension POSTs to ``https://admin.hlx.page/preview/<owner>/<repo>/main/<spreadsheet name>`` which is lacking the json extension due to the type detection not working (type is detected as _unknown_)

**Root cause**
Using hardcoded string detection for ``data-tooltip`` and ``aria-label`` does not work when using a non-English locale.

**Expected behavior**
Previewing from the folder view POSTs to ``https://admin.hlx.page/preview/<owner>/<repo>/main/<spreadsheet name>.json``

**Comment by rofe** [comment]
@cpiosecny thanks for reporting and debugging! A fix is being published (v6.45.1)

**Comment by cpiosecny** [comment]
Hey @rofe , thanks for the quick response but unfortunately this does not fix the underlying issue. 
Relevant DOM for spreadsheet named _bug_ with German Locale:

```
<div class="uXB7xe" jsaction=" mousedown:TiSivd;" aria-label="bug Freigegebene Google Tabellen-Datei">
    <div class="KL4NAf" data-tooltip="Google Tabellen: bug" aria-label="Google Tabellen: bug"
         data-tooltip-align="b,c" data-tooltip-delay="500" data-tooltip-unhoverable="true">bug
    </div>
</div>
```

I do not see how to match this properly across Locales without getting all translated Labels and matching against them :<
